share/spice/indego_bikeshare_locations/content.handlebars

```diff
@@ -0,0 +1,15 @@
+<div class="tile__body">
```

Comment:
The file path here needs to be changed to: `share/spice/bikesharing/phlindego/content.js`